docs/root/configuration/http_filters/cors_filter.rst

```diff
@@ -8,5 +8,5 @@ For the meaning of the headers please refer to the pages below.
 
 - https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS
 - https://www.w3.org/TR/cors/
-- :ref:`V1 API configuration <config_http_filters_cors_v1>`
-- :ref:`V2 API configuration <envoy_api_field_filter.network.HttpFilter.name>`
+- :ref:`v1 configuration <config_http_filters_cors_v1>`
```

Comment:
FWIW, I've been using "v1/v2 API reference" verbiage for links into the `api-v1/2/` trees from `configuration/`.

Reply:
I just want things to be consistent. I will switch it back and make it consistent.